exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -168,6 +169,12 @@ export default ExpFrameBaseComponent.extend({
         }
         return responses;
     }).volatile(),
+
+    allowNext: Ember.computed('cards.[]', function() {
+        console.log(this.get('cards').length);
+      return this.get('cards').length === 0 || !config.validate;
```

Comment:
This should return true if there are no cards left to sort, or if validation is turned off. I changed it to explicitly check `config.validate` for clarity.